lib/sarif/bandit_sarif.rb

```diff
@@ -0,0 +1,59 @@
+module Sarif
+  class BanditSarif < BaseSarif
+    BANDIT_URI = 'https://github.com/PyCQA/bandit'.freeze
+
+    def initialize(scan_report)
+      super(scan_report)
+      @uri = BANDIT_URI
+      @logs = parse_scan_report!
+      @unique_issues = Set.new
+    end
+
+    def parse_scan_report!
+      JSON.parse(@scan_report.log(''))['results']
+    rescue JSON::ParserError
+      []
+    end
+
+    def parse_issue(issue)
+      key = ""
+      key << issue["filename"] + issue["line_number"].to_s + issue['issue_text']
```

Comment:
You have similar code in the NPM file and I left a comment there, please take a look.